src/test/java/uk/gov/companieshouse/registeredemailaddressapi/integration/EligibilityControllerEmailMockingIntegrationTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.companieshouse.registeredemailaddressapi.integration;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MockMvc;
+
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.registeredemailaddressapi.integration.utils.Helper;
+import uk.gov.companieshouse.registeredemailaddressapi.service.CompanyProfileService;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestPropertySource(properties = {"env.name = livesbox"})
+class EligibilityControllerEmailMockingIntegrationTest {
+
+    Helper helper = new Helper();
```

Comment:
missing access specifier; could be "private static final HELPER = ..." and other occurrences; (in fact the Helper should be fixed such that all methods are statis and the class made final with a private constructor - but may be not your problem!)